123_state_tasks.R

```diff
@@ -1,18 +1,43 @@
-do_state_tasks <- function(oldest_active_sites) {
+do_state_tasks <- function(oldest_active_sites, ...) {
 
   # Define task table rows
-  # TODO: DEFINE A VECTOR OF TASK NAMES HERE
+  task_names <- unique(oldest_active_sites$state_cd)
 
   # Define task table columns
   download_step <- create_task_step(
-    step_name = 'download'
-    # TODO: Make target names like "WI_data"
-    # TODO: Make commands that call get_site_data()
+    step_name = 'download',
+    # Make target names like "WI_data"
+    target_name = function(task_name, ...) {
+      sprintf("%s_data", task_name)
+    },
+    # Make commands that call get_site_data()
+    command = function(task_name, ...) {
+      sprintf("get_site_data(sites_info = oldest_active_sites,
+                    state = I('%s'),
+                    parameter = parameter)", task_name)
```

Comment:
If you're looking for pretty indentation in the task-table remake file, you can use the `scipiper::psprintf()` function.